.ci/install_crio.sh

```diff
@@ -140,10 +140,15 @@ make BUILDTAGS="$(IFS=" "; echo "${build_union[*]}")"
 sudo -E install -D -m0755 runc "/usr/local/bin/crio-runc"
 popd
 
-echo "Set manage_network_ns_lifecycle to true"
-network_ns_flag="manage_network_ns_lifecycle"
+echo "Set manage_ns_lifecycle to true"
+network_ns_flag="manage_ns_lifecycle"
+# Set ns_network_flag for CRI-O versions less than 1.17
+crio_version_current=$(crio --version | egrep -o "[0-9]+\.[0-9]+\.[0-9]+" | head -1)
+if [ "$(compare_versions "$crio_version_current" "1.17.0")" -eq "1" ]; then
+	network_ns_flag="manage_network_ns_lifecycle"
+fi
 sudo sed -i "/\[crio.runtime\]/a$network_ns_flag = true" "$crio_config_file"
-sudo sed -i 's/manage_network_ns_lifecycle = false/#manage_network_ns_lifecycle = false/' "$crio_config_file"
+sudo sed -i "s/$network_ns_flag = false/#$network_ns_flag = false/" "$crio_config_file"
```

Comment:
this is not an option in cri-o 1.17. did you mean
`manage_ns_lifecycle`?

Reply:
oh jeez I see the piece above--we're setting a variable :upside_down_face:

Reply:
since the supported kubernetes versions upstream are all >= 1.17, and we switched to `manage_ns_lifecycle` starting in 1.17 [ref](https://github.com/cri-o/cri-o/releases/tag/v1.17.0)
 we  can switch to using `manage_ns_lifecycle` everywhere, and drop `manage_network_ns_lifecycle`

Reply:
+1, will make the change.

Reply:
included this in the drop-in config file and removed it from here.